packages/dolt/content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -1108,6 +1109,71 @@ WHERE staged=false;
 +------------+--------+-----------+
 ```
 
+## `dolt_workspace_$TABLENAME`
+
+This system table shows you which rows have been changed in your workspace and if they are staged.
```

Comment:
If we're going to use the term workspace, we should have docs for the concept, similar to working set:

https://docs.dolthub.com/concepts/dolt/git/working-set